Tests/test_Entrez_online.py

```diff
@@ -60,24 +60,24 @@ def test_parse_from_url(self):
         handle = Entrez.efetch(db='protein', id='15718680,157427902,119703751',
                                retmode='xml')
         self.assertTrue(handle.url.startswith(URL_HEAD + "efetch.fcgi?"), handle.url)
-        self.assertTrue(URL_TOOL in handle.url)
-        self.assertTrue(URL_EMAIL in handle.url)
-        self.assertTrue("id=15718680%2C157427902%2C119703751" in handle.url, handle.url)
+        self.assertIn(URL_TOOL, handle.url)
+        self.assertIn(URL_EMAIL, handle.url)
+        self.assertIn("id=15718680%2C157427902%2C119703751", handle.url)
         recs = list(Entrez.parse(handle))
         handle.close()
         self.assertEqual(3, len(recs))
         # arbitrary number, just to make sure the parser works
-        self.assertTrue(all(len(rec).keys > 5) for rec in recs)
+        self.assertIn(all(len(rec).keys > 5) for rec, recs)
```

Comment:
This is now giving a syntax error and causing TravisCI to fail the test. This is actually a generator expression inside an ``all(...)`` call.